File: 0684/sesión14 - consulta/SoftVen/SoftVenModel/TiendasDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftVen.Model
{
    public class TiendasDTO
    {
        private int? tiendaId;
        private string nombre;
        private string direccion;
        private bool? tiendaPrincipal;
        private double? comision;

        public TiendasDTO()
        {
            this.TiendaId = null;
            this.Nombre = null;
            this.Direccion = null;
            this.TiendaPrincipal = null;
            this.Comision = null;
        }

        public TiendasDTO(int tiendaId, string nombre, string direccion, bool tiendaPrincipal, double comision)
        {
            this.TiendaId = tiendaId;
            this.Nombre = nombre;
            this.Direccion = direccion;
            this.TiendaPrincipal = tiendaPrincipal;
            this.Comision = comision;
        }

        public int? TiendaId { get => tiendaId; set => tiendaId = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Direccion { get => direccion; set => direccion = value; }
        public bool? TiendaPrincipal { get => tiendaPrincipal; set => tiendaPrincipal = value; }
        public double? Comision { get => comision; set => comision = value; }
    }
}

File: 0684/sesión19 - WS - SOAP/Back-End/SoftVenModel/src/main/java/pe/edu/pucp/softven/model/TiendasDTO.java
package pe.edu.pucp.softven.model;

public class TiendasDTO {

    private Integer tiendaId;
    private String nombre;
    private String direccion;
    private Boolean tienda_principal;
    private Double comision;

    public TiendasDTO() {
        this.tiendaId = null;
        this.nombre = null;
        this.direccion = null;
        this.tienda_principal = null;
        this.comision = null;
    }

    public TiendasDTO(Integer tiendaId, String nombre, String direccion, Boolean tienda_principal, Double comision) {
        this.tiendaId = tiendaId;
        this.nombre = nombre;
        this.direccion = direccion;
        this.tienda_principal = tienda_principal;
        this.comision = comision;
    }

    /**
     * @return the tiendaId
     */
    public Integer getTiendaId() {
        return tiendaId;
    }

    /**
     * @param tiendaId the tiendaId to set
     */
    public void setTiendaId(Integer tiendaId) {
        this.tiendaId = tiendaId;
    }

    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the direccion
     */
    public String getDireccion() {
        return direccion;
    }

    /**
     * @param direccion the direccion to set
     */
    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    /**
     * @return the tienda_principal
     */
    public Boolean getTienda_principal() {
        return tienda_principal;
    }

    /**
     * @param tienda_principal the tienda_principal to set
     */
    public void setTienda_principal(Boolean tienda_principal) {
        this.tienda_principal = tienda_principal;
    }

    /**
     * @return the comision
     */
    public Double getComision() {
        return comision;
    }

    /**
     * @param comision the comision to set
     */
    public void setComision(Double comision) {
        this.comision = comision;
    }

}
